keanu-project/src/main/java/io/improbable/keanu/distributions/continuous/Gamma.java

```diff
@@ -18,123 +18,124 @@
 import io.improbable.keanu.tensor.dbl.DoubleTensor;
 import io.improbable.keanu.vertices.dbl.KeanuRandom;
 
+/**
+ * @see "Computer Generation of Statistical Distributions
+ * by Richard Saucier,
+ * ARL-TR-2168 March 2000,
+ * 5.1.11 page 23"
+ */
 public class Gamma implements ContinuousDistribution {
 
     private static final double M_E = 0.577215664901532860606512090082;
     private final DoubleTensor location;
```

Comment:
@gordoncaleb removed `location` from the `Exponential` distribution - do we want to remove it from here too?

Reply:
#173